templates/repo/diff/box.tmpl

```diff
@@ -31,12 +31,57 @@
 			{{end}}
 			{{template "repo/diff/whitespace_dropdown" .}}
 			{{template "repo/diff/options_dropdown" .}}
+			{{if .PageIsPullFiles}}
+				<script id="commit-data-script">
+					{{$numCommits := len .Commits}}
+					window.config.pageData.commitInfo = {
+						issueLink: {{$.Issue.Link}},
+						lastReviewCommitSha: {{.LastReviewSha}},
+						commits: [
+						{{range .Commits}}
+								{
+									summary: {{.Summary}},
+									committerOrAuthorName: {{if .Committer}}{{.Committer.Name}}{{else}}{{.Author.Name}}{{end}},
+									id: {{.ID.String}},
+									time: {{if .Committer}}{{.Committer.When}}{{else}}{{.Author.When}}{{end}},
+									selected: false
+								},
+						{{end}}],
+						locale: {
+							lang: {{$.locale}},
```

Comment:
`{{$.locale}}` doesn't seem right

Reply:
fixed now - needed info is loaded via fetch